### PR TITLE
Fixed #3224 leaving example code blocks intact

### LINKED ISSUE #3224: Code snippets in developer docs should work when copied
**Describe the bug**
Most code snippets in the developer docs (as opposed to all other docs) are prefixed with `$`, indicating they are supposed to be executed on the command line.

**Product**
FlowKit main docs

**Version**
ea012c1fa5efcf7b80a1a1da7de9e0fc54bd5621

**To Reproduce**
Go to https://flowkit.xyz/developer/dev_environment_setup/, copy one of the commands and paste into your command line.

**Expected behavior**
When pasting, it should not have a leading `$` as this makes the command fail.

**Additional context**
The `$` indicates it is a command and makes sense to be contained in longer code snippets that consist of multiple commands where it acts as a visual indicator/separator for individual commands.

### PR DESCRIPTION
**Pull request by flowstef**
Closes #3224

### I have:

- [x] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [x] Added tests for any new additions
- [x] Added or updated any relevant documentation
- [x] Added an Architectural Decision Record (ADR), if appropriate
- [x] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [x] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

removed `$` from code snipptes in developer docs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 ### Fixed
+- removed `$` from code snipptes in developer docs [#3224](https://github.com/Flowminder/FlowKit/issues/3224)
```
Comment (greenape) [inline CHANGELOG.md:14]:
```suggestion
- Removed `$` from code snippets in developer docs [#3224](https://github.com/Flowminder/FlowKit/issues/3224)
```

**Review by greenape — CHANGES_REQUESTED** [review]
One typo, otherwise great - thanks 👍

File: CHANGELOG.md (hunk 1 above)
Comment (flowstef) [reply CHANGELOG.md:14]: Oops!

**Comment by flowstef** [comment]
How did the build fail when all I changed was a typo in the changelog?🧐

**Comment by greenape** [comment]
Looks like respx had a new release a few hours ago - the FlowClient tests on travis aren't pinned, and the update to that has caused tests to fail.

**Comment by greenape** [comment]
#3240 should fix the fail once merged to master.

**Review by greenape — APPROVED** [review]

**Pull request merged**